### PR TITLE
Bump ui-component

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
n/a
linked to https://github.com/gravitee-io/gravitee-ui-components/pull/509 and https://github.com/gravitee-io/gravitee-ui-components/pull/512
**Description**

Bump ui-components into portal et console

**Additional context**

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-mkqnmedkej.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/bump-ui-compo/index.html)
_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

### CODE REVIEW

File: gravitee-apim-portal-webui/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "prettier:fix": "prettier --write \"**/*.{js,ts,html,css,scss,json}\"",
     "prettier:list": "prettier --list-different \"**/*.{js,ts,html,css,scss,json}\"",
     "serve": "ng serve --port=4100",
+    "serve:apim-master": "ng run gravitee-portal-webui:serve:apim-master --port=4100",
```
Comment (gaetanmaisse) [inline gravitee-apim-portal-webui/package.json]: Not sure about this

File: gravitee-apim-console-webui/package-lock.json [hunk 2]
```diff
@@ -8775,9 +8775,9 @@
           }
         },
         "codemirror": {
-          "version": "5.63.3",
-          "resolved": "https://registry.npmjs.org/codemirror/-/codemirror-5.63.3.tgz",
-          "integrity": "sha512-1C+LELr+5grgJYqwZKqxrcbPsHFHapVaVAloBsFBASbpLnQqLw1U8yXJ3gT5D+rhxIiSpo+kTqN+hQ+9ialIXw=="
+          "version": "5.64.0",
+          "resolved": "https://registry.npmjs.org/codemirror/-/codemirror-5.64.0.tgz",
```
Comment (gcusnieux) [inline gravitee-apim-console-webui/package-lock.json:8779]: Stange... ui-components use codemirror 5.63

**Review by gcusnieux — APPROVED** [review]

File: gravitee-apim-portal-webui/package.json (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-portal-webui/package.json]: Oups

File: gravitee-apim-console-webui/package-lock.json (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/package-lock.json:8779]: yes but the ui-components dependencies is `"codemirror": "^5.63.2",` so minor increment is ok, i think 🤔

File: gravitee-apim-console-webui/package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
     "@angular/upgrade": "12.2.3",
     "@asyncapi/web-component": "1.0.0-next.15",
     "@fontsource/libre-franklin": "4.4.5",
-    "@gravitee/ui-components": "3.22.0",
+    "@gravitee/ui-components": "^3.24.4",
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/package.json]: you should use pinned version

File: gravitee-apim-portal-webui/package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "@angular/router": "9.1.1",
     "@asyncapi/web-component": "1.0.0-next.15",
     "@formatjs/intl-relativetimeformat": "9.3.1",
-    "@gravitee/ui-components": "3.23.0",
+    "@gravitee/ui-components": "^3.24.4",
```
Comment (gaetanmaisse) [inline gravitee-apim-portal-webui/package.json]: same here

File: gravitee-apim-console-webui/package.json (hunk 3 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/package.json]: yes, I don't understand why npm is adding it back  like this

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**